#include <stdio.h>

#include "throw.h"

int main() {
  test_exception();
  printf("after test_exception()\n");
  return 0;
}